SU2_PY/SU2/eval/design.py

```diff
@@ -122,8 +122,15 @@ def _eval(self,eval_func,*args):
         # check folder
         assert os.path.exists(folder) , 'cannot find design folder %s' % folder
         
+        konfig = copy.deepcopy(config)
+
+        if 'TIME_ITER' in state.WND_CAUCHY_DATA:         # Use Convergence data, if we have already a direct run
+            konfig['TIME_ITER'] = state.WND_CAUCHY_DATA['TIME_ITER']
+            konfig['ITER_AVERAGE_OBJ'] = state.WND_CAUCHY_DATA['ITER_AVERAGE_OBJ']
+            konfig['UNST_ADJOINT_ITER'] = state.WND_CAUCHY_DATA['UNST_ADJOINT_ITER']
+
```

Comment:
I am not familiar with the python framework, but I feel this needs a comment.

Reply:
If the windowed time convergence criterion is satisfied, the solver may terminate before it has reached the iteration number TIME_ITER. In this case values of TIME_ITER, ITER_AVERAGE_OBJ and UNST_ADJOINT_ITER must be updated appropriately (this happens in /run/direct.py line 120-127). The code section above updates the configuration file for the adjoint run. 
Indeed this is not obvious and I have added comments to clearify this.